app/controllers/workflows_controller.rb

```diff
@@ -79,7 +79,6 @@ def create
         version: params[:version]
       )
     ).create_workflow_steps
-    SendUpdateMessage.publish(druid: params[:druid])
```

Comment:
This moved to the WorkflowCreator? Can you say more about this change?

Reply:
I believe this to be unnecessary, because `WorkflowCreator#create_workflow_steps` calls `WorkflowCreator#enqueue` which calls `SendUpdateMessage.publish`

Reply:
OK.